package.json

```diff
@@ -5,19 +5,33 @@
   "scripts": {
     "build": "tsc",
     "test": "mocha",
-    "lint": "tslint '*(test|src)/**/*.ts'"
+    "lint": "tslint '*(test|src)/**/*.ts'",
+    "start": "tsc && node dist/index.js"
   },
   "devDependencies": {
     "@types/chai": "4.2.3",
     "@types/mocha": "5.2.7",
-    "chai": "4.2.0",
-    "mocha": "6.2.0",
+    "@types/express": "^4.17.1",
```

Comment:
I prefer to avoid using `^` because it automatically updates the bugfix version and you never know if the subpackage will release a broken version or introduce breaking changes in a bugfix release